go/vt/vtgate/planbuilder/testdata/union_cases.txt

```diff
@@ -279,4 +279,3 @@
     ]
   }
 }
-
```

Comment:
reset this file change.

Reply:
Hmm... Weird. On my machine, with `723fef02e` checked out, that file has an extra line at the end, and git does not see any changes. Not sure what's going on...